src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -333,6 +340,48 @@ ov::Any Plugin::get_property(const std::string& name, const ov::AnyMap& options)
         return decltype(ov::internal::caching_properties)::value_type(get_caching_properties());
     }
 
+    // Below properties depend on the device ID.
+    if (name == ov::internal::compiled_model_format.name()) {
+        auto model_format_map = m_compiled_model_format;
+        OPENVINO_ASSERT(m_device_map.find(m_compiled_model_format_device_id) != m_device_map.end(),
+                        "[GPU] compiled_model_format: Couldn't find device for GPU with id ",
+                        m_compiled_model_format_device_id);
+        // Set specified device info for compiled_model_format
+        model_format_map["DEVICE_ID"] = m_compiled_model_format_device_id;
+        model_format_map["DRIVER_VERSION"] =
+            m_device_map.at(m_compiled_model_format_device_id)->get_info().driver_version;
```

Comment:
but as we saw, driver version does not depend on actual device, because when you install driver on the system, it works for all devices.

Reply:
You mean we need bind to the actual device, not driver version?

Reply:
Actual device can be represented by `driver_info.vendor_id + driver_info.device_id` ?
So that we apply below 3 items:
1.  vendor_id
2.  device_id
3. driver_version

Does it make sense? @ilya-lavrenov

Reply:
Please, have a look at https://github.com/openvinotoolkit/openvino/pull/21492/files#r1426253560
IMO we need to cover only driver version here, because device architecture is already a part of caching_properties.